airflow/plugins/operators/__init__.py

```diff
@@ -1,9 +1,11 @@
 # flake8: noqa
-from .external_table import ExternalTable
-from .once_off_external_task_sensor import OnceOffExternalTaskSensor
-from .pod_operator import PodOperator
-from .python_taskflow_operator import PythonTaskflowOperator
-from .sql_to_warehouse_operator import SqlToWarehouseOperator
-from .csv_to_warehouse_operator import CsvToWarehouseOperator
-from .python_to_warehouse_operator import PythonToWarehouseOperator
-from .sql_query_operator import SqlQueryOperator
+# TODO: after upgrading airflow, using a relative import raises the error
+# "attempted relative import with no known parent package"
```

Comment:
This seems more explicit anyways. Can this comment be removed now?

Reply:
Will remove the comment! It's probably won't be an issue, but the challenge with this format is that it's ambiguous where they're being imported from. It could be either an installed (or the built in) operators package, or this operators folder. The dots are unambiguous that it's from our internal folder. (but I'm not sure why this error is occurring :/)